app/javascript/pages/country/embed/embed.js

```diff
@@ -0,0 +1,19 @@
+import { createElement, PureComponent } from 'react';
+import { connect } from 'react-redux';
+
+import EmbedComponent from './embed-component';
+
+const mapStateToProps = ({ location, countryData }) => ({
+  widgetKey: location.payload.widget,
+  isGeostoreLoading: countryData.isGeostoreLoading
```

Comment:
This is the only key that has no relevance to the widgets for loading. It is the country and region data we need often for the titles.

Reply:
But some widgets are using locationNames, so we need to wait to `isGeostoreLoading = true`, right?

Reply:
geostore is only bound to the map so we dont need to wait for it. Countries, Regions, and SubRegions are what contains the data.